### PR TITLE
Feature/burbano dawson

### PR DESCRIPTION
**Pull request by william-dawson**
I have added a description of the collaboration between Mario Burbano and myself. I also added Mario to the list of people in the data directory.

### CODE REVIEW

File: _projects/ewald_matrix.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+layout: page_project
+title: Joint Investigation Into the Structure of Ewald Matrices
+date: 2017-12-13
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics:
+  - numerics
```
Comment (pancetta) [inline _projects/ewald_matrix.md]: Doesn't this fit into applications as well?

File: _projects/ewald_matrix.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+{% bibliography --cited --file jlesc.bib %}
+
+
+## Future plans
+We are continuing to work on generating representative matrices and are performing 
+some preliminary calculations.
+
+## References
+
+{% bibliography --file external/ewald_matrix %}
```
Comment (pancetta) [inline _projects/ewald_matrix.md]: Could you cite the reference somewhere in the text?

**Review by pancetta — CHANGES_REQUESTED** [review]
Well done! Nice to see someone doing this procedure without bazillion mehmehmeh-emails.. only minor comments/questions, see below.

File: _projects/ewald_matrix.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+layout: page_project
+title: Joint Investigation Into the Structure of Ewald Matrices
+date: 2017-12-13
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
```
Comment (pancetta) [inline _projects/ewald_matrix.md]: Is it really running or is it starting?

**Comment by william-dawson** [comment]
Thanks for the feedback Robert, I've made modifications in accordance to your suggestions. Let me know if there is anything else you recommend!

**Comment by pancetta** [comment]
Nope, all good! Thanks!

**Pull request merged**